src/twisted/conch/ssh/keys.py

```diff
@@ -277,13 +277,16 @@ def _fromString_BLOB(cls, blob):
                 curve=b"ecdsa-sha2-nistp256",
             )
             keyObject._sk = True
+            keyObject.application = common.getNS(rest, 3)[2]
```

Comment:
Very minor change to avoid unwrapping multiple times
For line 276
```
_, encodedPoint, rest = common.getNS(rest, 2),
```

Then have something similar to `ssh-ed25519` handling
```suggestion
            keyObject.application = common.getNS(rest)[0]
```